tests/cross_functional/ui/fusion_access/test_virtualmachine_lifecycle.py

```
@@ -59,6 +59,83 @@ def setup_ui(self, setup_ui_class_factory):
+    def teardown_lungroup(self, request):
+        """
+        Class-scoped teardown that runs once after all tests in this class.
+
+        Steps:
+        1. Delete the LUN group via the UI
+        2. Wait 60 s then delete the LocalDisk associated with the LUN group
+           from the CLI.
+        3. Wait 60 s then delete the IBM Spectrum Scale cluster resource.
+        """
+
+        def cleanup():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `cleanup`

Add return type annotation: `None`

(ANN202)